app/templates/components/delivery-table-row.hbs

```diff
@@ -1,6 +1,9 @@
-<td>{{ delivery.id }}</td>
-<td>{{ delivery.project.name }}</td>
-<td>{{ delivery.fromUser.fullName }}</td>
-<td>{{ delivery.toUser.fullName}}</td>
-<td>{{ delivery.state }}</td>
-<td>{{ delivery.transferId }}</td>
+<td>{{#link-to "deliveries.show" transfer}}{{transfer.id}}{{/link-to}}</td>
```

Comment:
Showing the transfer UUID as the link text isn't terribly useful. If we don't have a better field to use as the link text, maybe we should use the project name?